kubeflow/rebuild-image.sh

```diff
@@ -34,3 +41,5 @@ if test -n "${JUPYTER_IMAGE}" ; then
 fi
 
 rm lib.sh
+
+docker system prune -f
```

Comment:
You cannot have this unconditionally on every build because it's a potentially destructive action (it will remove stopped containers). Run it yourself if you need it.